chapters/functions.tex

```diff
@@ -1735,20 +1740,21 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 \end{annotationdefinition}
 
 \begin{annotationdefinition}[InlineAfterIndexReduction]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"InlineAfterIndexReduction" "=" ( false | true )
+\begin{synopsis}\begin{lstlisting}
+/*literal*/ constant Boolean InlineAfterIndexReduction;
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
 Has only an effect within a function declaration.
 
 If {\lstinline!true!}, the model developer proposes to inline the function after the function is differentiated for index reduction, and before any other symbolic transformations are performed.
 This annotation cannot be combined with annotations {\lstinline!Inline!} and {\lstinline!LateInline!}.
+The default is to not perform this specific inlining.
```

Comment:
How about swapping order with the previous sentence, and then have a paragraph break before the rule about combining annotations?

Reply:
Should be done now.

Reply:
Looks fine.  Resolving.